schemas/questionnaire_v1.json

```diff
@@ -62,30 +62,28 @@
       ]
     },
     "metadata": {
```

Comment:
Is asserting the presence of `user_id` and `period_id` required as before? from a quick look at json schema, it looks like moving the checks to `_validate_schema_contain_metadata` is the only reasonable way to handle that check now.